<html xmlns:v="urn:schemas-microsoft-com:vml"xmlns:o="urn:schemas-microsoft-com:office:office"xmlns:w="urn:schemas-microsoft-com:office:word"xmlns:p="urn:schemas-microsoft-com:office:powerpoint"xmlns:oa="urn:schemas-microsoft-com:office:activation"xmlns:st1="urn:schemas-microsoft-com:office:smarttags"xmlns="http://www.w3.org/TR/REC-html40"><head><meta http-equiv=Content-Type content="text/html; charset=iso-8859-1"><meta name=ProgId content=Word.Document><meta name=Generator content="Microsoft Word 11"><meta name=Originator content="Microsoft Word 11"><link rel=File-List href="B_Deagle%202010_files/filelist.xml"><link rel=Edit-Time-Data href="B_Deagle%202010_files/editdata.mso"><!--[if !mso]><style>v\:* {behavior:url(#default#VML);}o\:* {behavior:url(#default#VML);}w\:* {behavior:url(#default#VML);}.shape {behavior:url(#default#VML);}</style><![endif]--><title>Bruce Deagle</title><o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags" name="place"/><o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags" name="PlaceType"/><o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags" name="PlaceName"/><!--[if gte mso 9]><xml> <o:DocumentProperties>  <o:Author>Tom</o:Author>  <o:Template>Normal</o:Template>  <o:LastAuthor> BD</o:LastAuthor>  <o:Revision>14</o:Revision>  <o:TotalTime>737</o:TotalTime>  <o:Created>2010-11-05T01:02:00Z</o:Created>  <o:LastSaved>2010-11-08T06:43:00Z</o:LastSaved>  <o:Pages>1</o:Pages>  <o:Words>1237</o:Words>  <o:Characters>7055</o:Characters>  <o:Lines>58</o:Lines>  <o:Paragraphs>16</o:Paragraphs>  <o:CharactersWithSpaces>8276</o:CharactersWithSpaces>  <o:Version>11.6568</o:Version> </o:DocumentProperties></xml><![endif]--><!--[if gte mso 9]><xml> <w:WordDocument>  <w:ValidateAgainstSchemas/>  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>  <w:BrowserLevel>MicrosoftInternetExplorer4</w:BrowserLevel> </w:WordDocument></xml><![endif]--><!--[if gte mso 9]><xml> <w:LatentStyles DefLockedState="false" LatentStyleCount="156"> </w:LatentStyles></xml><![endif]--><!--[if !mso]><object classid="clsid:38481807-CA0E-42D2-BF39-B33AF135CC4D" id=ieooui></object><style>st1\:*{behavior:url(#ieooui) }</style><![endif]--><style><!-- /* Font Definitions */ @font-face	{font-family:Helvetica;	panose-1:2 11 6 4 2 2 2 2 2 4;	mso-font-charset:0;	mso-generic-font-family:swiss;	mso-font-pitch:variable;	mso-font-signature:536902279 -2147483648 8 0 511 0;}@font-face	{font-family:TimesNewRoman;	panose-1:0 0 0 0 0 0 0 0 0 0;	mso-font-charset:0;	mso-generic-font-family:roman;	mso-font-format:other;	mso-font-pitch:auto;	mso-font-signature:3 0 0 0 1 0;}@font-face	{font-family:Verdana;	panose-1:2 11 6 4 3 5 4 4 2 4;	mso-font-charset:0;	mso-generic-font-family:swiss;	mso-font-pitch:variable;	mso-font-signature:536871559 0 0 0 415 0;}@font-face	{font-family:Georgia;	panose-1:2 4 5 2 5 4 5 2 3 3;	mso-font-charset:0;	mso-generic-font-family:roman;	mso-font-pitch:variable;	mso-font-signature:647 0 0 0 159 0;} /* Style Definitions */ p.MsoNormal, li.MsoNormal, div.MsoNormal	{mso-style-parent:"";	margin:0cm;	margin-bottom:.0001pt;	mso-pagination:widow-orphan;	font-size:12.0pt;	font-family:"Times New Roman";	mso-fareast-font-family:"Times New Roman";	color:#FFFFCC;}a:link, span.MsoHyperlink	{color:#FFCC00;	text-decoration:underline;	text-underline:single;}a:visited, span.MsoHyperlinkFollowed	{color:#FFFF99;	text-decoration:underline;	text-underline:single;}p	{mso-margin-top-alt:auto;	margin-right:0cm;	mso-margin-bottom-alt:auto;	margin-left:0cm;	mso-pagination:widow-orphan;	font-size:12.0pt;	font-family:"Times New Roman";	mso-fareast-font-family:"Times New Roman";	color:#FFFFCC;}@page Section1	{size:612.0pt 792.0pt;	margin:72.0pt 90.0pt 72.0pt 36.0pt;	mso-header-margin:36.0pt;	mso-footer-margin:36.0pt;	mso-paper-source:0;}div.Section1	{page:Section1;} /* List Definitions */ @list l0	{mso-list-id:1867134800;	mso-list-type:hybrid;	mso-list-template-ids:1204211534 67698689 67698691 67698693 67698689 67698691 67698693 67698689 67698691 67698693;}@list l0:level1	{mso-level-number-format:bullet;	mso-level-text:\F0B7;	mso-level-tab-stop:18.0pt;	mso-level-number-position:left;	margin-left:18.0pt;	text-indent:-18.0pt;	font-family:Symbol;}@list l0:level2	{mso-level-tab-stop:72.0pt;	mso-level-number-position:left;	text-indent:-18.0pt;}@list l0:level3	{mso-level-tab-stop:108.0pt;	mso-level-number-position:left;	text-indent:-18.0pt;}@list l0:level4	{mso-level-tab-stop:144.0pt;	mso-level-number-position:left;	text-indent:-18.0pt;}@list l0:level5	{mso-level-tab-stop:180.0pt;	mso-level-number-position:left;	text-indent:-18.0pt;}@list l0:level6	{mso-level-tab-stop:216.0pt;	mso-level-number-position:left;	text-indent:-18.0pt;}@list l0:level7	{mso-level-tab-stop:252.0pt;	mso-level-number-position:left;	text-indent:-18.0pt;}@list l0:level8	{mso-level-tab-stop:288.0pt;	mso-level-number-position:left;	text-indent:-18.0pt;}@list l0:level9	{mso-level-tab-stop:324.0pt;	mso-level-number-position:left;	text-indent:-18.0pt;}ol	{margin-bottom:0cm;}ul	{margin-bottom:0cm;}--></style><!--[if gte mso 10]><style> /* Style Definitions */ table.MsoNormalTable	{mso-style-name:"Table Normal";	mso-tstyle-rowband-size:0;	mso-tstyle-colband-size:0;	mso-style-noshow:yes;	mso-style-parent:"";	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;	mso-para-margin:0cm;	mso-para-margin-bottom:.0001pt;	mso-pagination:widow-orphan;	font-size:10.0pt;	font-family:"Times New Roman";	mso-ansi-language:#0400;	mso-fareast-language:#0400;	mso-bidi-language:#0400;}</style><![endif]--><meta name=Author content="Carolyn Bergstrom"><script language="JavaScript"><!--function MM_preloadImages() { //v3.0  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}}function MM_findObj(n, d) { //v4.0  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);  if(!x && document.getElementById) x=document.getElementById(n); return x;}function MM_nbGroup(event, grpName) { //v3.0  var i,img,nbArr,args=MM_nbGroup.arguments;  if (event == "init" && args.length > 2) {    if ((img = MM_findObj(args[2])) != null && !img.MM_init) {      img.MM_init = true; img.MM_up = args[3]; img.MM_dn = img.src;      if ((nbArr = document[grpName]) == null) nbArr = document[grpName] = new Array();      nbArr[nbArr.length] = img;      for (i=4; i < args.length-1; i+=2) if ((img = MM_findObj(args[i])) != null) {        if (!img.MM_up) img.MM_up = img.src;        img.src = img.MM_dn = args[i+1];        nbArr[nbArr.length] = img;    } }  } else if (event == "over") {    document.MM_nbOver = nbArr = new Array();    for (i=1; i < args.length-1; i+=3) if ((img = MM_findObj(args[i])) != null) {      if (!img.MM_up) img.MM_up = img.src;      img.src = (img.MM_dn && args[i+2]) ? args[i+2] : args[i+1];      nbArr[nbArr.length] = img;    }  } else if (event == "out" ) {    for (i=0; i < document.MM_nbOver.length; i++) {      img = document.MM_nbOver[i]; img.src = (img.MM_dn) ? img.MM_dn : img.MM_up; }  } else if (event == "down") {    if ((nbArr = document[grpName]) != null)      for (i=0; i < nbArr.length; i++) { img=nbArr[i]; img.src = img.MM_up; img.MM_dn = 0; }    document[grpName] = nbArr = new Array();    for (i=2; i < args.length-1; i+=2) if ((img = MM_findObj(args[i])) != null) {      if (!img.MM_up) img.MM_up = img.src;      img.src = img.MM_dn = args[i+1];      nbArr[nbArr.length] = img;  } }}//--></script><!--[if gte mso 9]><xml> <o:shapedefaults v:ext="edit" spidmax="43010" style='mso-wrap-style:none;  v-text-anchor:top-baseline' fill="f" fillcolor="#bbe0e3" stroke="f">  <v:fill color="#bbe0e3" on="f"/>  <v:stroke on="f"/>  <v:textbox style='mso-fit-shape-to-text:t' inset=".91439mm,.45719mm,.91439mm,.45719mm"/>  <o:colormenu v:ext="edit" fillcolor="none" strokecolor="red"/> </o:shapedefaults></xml><![endif]--><!--[if gte mso 9]><xml> <o:shapelayout v:ext="edit">  <o:idmap v:ext="edit" data="1"/> </o:shapelayout></xml><![endif]--></head><body bgcolor="#000066" lang=EN-AU link="#FFCC00" vlink="#FFFF99"style='tab-interval:36.0pt' alink="#cc9933"onload="MM_preloadImages('Home1.JPG','Home2.JPG','adap1.JPG','adap2.JPG','molphy1.JPG','molphy2.JPG','nutcyc1.JPG','nutcyc2.JPG','lab1.JPG','lab2.JPG','Public1.JPG','Public2.JPG')"><div class=Section1><p class=MsoNormal>&nbsp; <o:p></o:p></p><div align=center><table class=MsoNormalTable border=0 cellspacing=0 cellpadding=0 width="100%" style='width:100.0%;mso-cellspacing:0cm;mso-padding-alt:0cm 0cm 0cm 0cm' height=67> <tr style='mso-yfti-irow:0;mso-yfti-firstrow:yes;mso-yfti-lastrow:yes'>  <td width="16%" style='width:16.0%;padding:0cm 0cm 0cm 0cm'>  <p class=MsoNormal align=center style='text-align:center'><b><a  href="../index.html"><span style='text-decoration:none;text-underline:none'><img  border=0 width=157 height=113 id="_x0000_i1030" src="../home.jpg"  alt="Home Page"></span></a><o:p></o:p></b></p>  </td>  <td width="16%" style='width:16.0%;padding:0cm 0cm 0cm 0cm'>  <p class=MsoNormal align=center style='text-align:center'><b><a  href="../Functmorph.html"><span style='text-decoration:none;text-underline:  none'><img border=0 width=157 height=113 id="_x0000_i1031"  src="../adaptive%20radiation.jpg"  alt="Adaptive Radiation and Molecular Studies"></span></a><o:p></o:p></b></p>  </td>  <td width="16%" style='width:16.0%;padding:0cm 0cm 0cm 0cm'>  <p class=MsoNormal align=center style='text-align:center'><b><a  href="../nutrientcycle2.html"><span style='text-decoration:none;text-underline:  none'><img border=0 width=157 height=113 id="_x0000_i1032"  src="../salmonforestproject.jpg" alt="The Salmon Forest Project"></span></a><o:p></o:p></b></p>  </td>  <td width="16%" style='width:16.0%;padding:0cm 0cm 0cm 0cm'>  <p class=MsoNormal align=center style='text-align:center'><b><a  href="../molecular.html"><span style='text-decoration:none;text-underline:  none'><img border=0 width=157 height=113 id="_x0000_i1033"  src="../molecular%20studies.jpg" alt="Molecular Studies"></span></a><o:p></o:p></b></p>  </td>  <td width="16%" style='width:16.0%;padding:0cm 0cm 0cm 0cm'>  <p class=MsoNormal align=center style='text-align:center'><b><a  href="../Thelab.html"><span style='text-decoration:none;text-underline:none'><img  border=0 width=157 height=113 id="_x0000_i1034" src="../lab%20members.jpg"  alt="Members of the lab"></span></a><o:p></o:p></b></p>  </td>  <td width="16%" style='width:16.0%;padding:0cm 0cm 0cm 0cm'>  <p class=MsoNormal align=center style='text-align:center'><b><a  href="../Publications.html"><span style='text-decoration:none;text-underline:  none'><img border=0 width=157 height=113 id="_x0000_i1035"  src="../publications.jpg" alt=Publications></span></a><o:p></o:p></b></p>  </td> </tr></table></div><p align=center style='text-align:center'><b><span style='font-size:22.0pt;font-family:Georgia'><o:p>&nbsp;</o:p></span></b></p><p align=center style='text-align:center'><b><span style='font-size:20.0pt;mso-bidi-font-size:22.0pt;font-family:Georgia'>&nbsp;</span></b><b><spanstyle='font-size:20.0pt;mso-bidi-font-size:22.0pt;font-family:Georgia;mso-bidi-font-family:Helvetica'>Bruce<span style='mso-spacerun:yes'>�</span>Deagle, Post-Doc</span></b><span style='font-size:20.0pt;mso-bidi-font-size:22.0pt;font-family:Georgia;mso-bidi-font-family:Helvetica'><br><b>email: <a href="mailto:bdeagle@uvic.ca">bdeagle@uvic.ca<spanstyle='mso-bidi-font-family:"Times New Roman"'><br></span></a><o:p></o:p></b></span></p><p class=MsoNormal><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia'>The focus of my research has been using information in DNAto address ecological and evolutionary questions. I am currently working on a sticklebackpopulation genomics project in Tom Reimchen&#8217;s laboratory; we are workingin close collaboration with </span><span style='font-size:14.0pt;font-family:Georgia'><a href="http://kingsley.stanford.edu/">David Kingsley&#8217;s group</a></span><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia'>at <st1:place w:st="on"><st1:PlaceName w:st="on">Stanford</st1:PlaceName> <st1:PlaceType w:st="on">University</st1:PlaceType></st1:place>. During my PhD,I looked at methods to study the diet of large marine predators by analysingprey DNA in their faeces.<o:p></o:p></span></p><p class=MsoNormal><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div class=MsoNormal><span style='font-size:11.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia;color:windowtext'><hr size=2 width="100%" align=left></span></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div><p class=MsoNormal align=center style='mso-margin-top-alt:auto;margin-bottom:12.0pt;text-align:center'><u><span style='font-size:20.0pt;font-family:Georgia'>Postdoctoralwork: SNP variation in Haida Gwaii threespine stickleback</span></u><spanstyle='font-size:20.0pt;font-family:Georgia'><span style='mso-spacerun:yes'>���</span><o:p></o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><a name="OLE_LINK1"></a><a name="OLE_LINK2"><spanstyle='mso-bookmark:OLE_LINK1'><span lang=EN-US style='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'>Studies of Haida Gwaii sticklebackover the last 35 years have revealed a remarkable level of phenotypic variationwithin and among freshwater populations, providing a classic example ofadaptive radiation </span></span></a><span style='mso-bookmark:OLE_LINK2'><spanstyle='mso-bookmark:OLE_LINK1'><span style='font-size:14.0pt;font-family:Georgia'>(</span></span></span><ahref="http://web.uvic.ca/~reimlab/Publications.html"><span style='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><span style='font-size:14.0pt;font-family:Georgia'>see Reimchen publications</span></span></span><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'></span></span></a><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><spanstyle='font-size:14.0pt;font-family:Georgia'>)</span></span></span><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><spanlang=EN-US style='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'>.</span></span></span><span style='mso-bookmark:OLE_LINK2'><spanstyle='mso-bookmark:OLE_LINK1'><span style='font-size:14.0pt;font-family:Georgia'>Withrecent completion of a high-quality reference genome sequence, the threespinestickleback is also quickly becoming an important model in the field ofevolutionary genomics (</span></span></span><ahref="http://kingsley.stanford.edu/"><span style='mso-bookmark:OLE_LINK2'><spanstyle='mso-bookmark:OLE_LINK1'><span style='font-size:14.0pt;font-family:Georgia'>seeKingsley publications</span></span></span><span style='mso-bookmark:OLE_LINK2'><spanstyle='mso-bookmark:OLE_LINK1'></span></span></a><span style='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><span lang=EN-USstyle='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'>)</span></span></span><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><spanstyle='font-size:14.0pt;font-family:Georgia'>.</span></span></span><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><spanstyle='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'> </span></span></span><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><spanstyle='font-size:14.0pt;font-family:Georgia'>In this postdoctoral project weare obtaining a genomic perspective of the genetic structure and history of HaidaGwaii stickleback using a genome-wide SNP (single nucleotide polymorphisms)genotyping system </span></span></span><span style='mso-bookmark:OLE_LINK2'><spanstyle='mso-bookmark:OLE_LINK1'><span lang=EN-US style='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'>(Illumina� GoldenGate) </span></span></span><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><spanstyle='font-size:14.0pt;font-family:Georgia'>developed at Stanford (Figure1).<span style='mso-spacerun:yes'>� </span>Particular areas of interest include:<o:p></o:p></span></span></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt'><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><b><spanstyle='font-size:14.0pt;font-family:Georgia'><spanstyle='mso-spacerun:yes'>���������������������� </span>(A)</span></b></span></span><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><spanstyle='font-size:14.0pt;font-family:Georgia'> <b style='mso-bidi-font-weight:normal'>Phylogeography of the Haida Gwaii freshwater populations</b><o:p></o:p></span></span></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;tab-stops:center 27.0pt'><span style='mso-bookmark:OLE_LINK2'><spanstyle='mso-bookmark:OLE_LINK1'><span style='font-size:14.0pt;font-family:Georgia'><spanstyle='mso-spacerun:yes'>���������������������� </span><b>(B)</b> </span></span></span><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><bstyle='mso-bidi-font-weight:normal'><span lang=EN-US style='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'>The population genomics ofparapatric stream/lake pairs of stickleback</span></b></span></span><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><spanstyle='font-size:14.0pt;font-family:Georgia'><o:p></o:p></span></span></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt'><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><spanlang=EN-US style='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'><spanstyle='mso-spacerun:yes'>��������������������� </span><b>(C) </b></span></span></span><spanstyle='mso-bookmark:OLE_LINK2'><span style='mso-bookmark:OLE_LINK1'><bstyle='mso-bidi-font-weight:normal'><span style='font-size:14.0pt;font-family:Georgia'>Associations between genetic markers and particular morphologicaltraits</span></b></span></span><span style='mso-bookmark:OLE_LINK1'></span><spanstyle='mso-bookmark:OLE_LINK2'></span><span style='font-size:14.0pt;font-family:Georgia'><o:p></o:p></span></p><div style='margin-top:5.0pt;margin-bottom:12.0pt'><div><div><div><div><div><div><div><div><div><div><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><b style='mso-bidi-font-weight:normal'><span lang=EN-USstyle='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></b></p><p class=MsoNormal align=center style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt;text-align:center'><b style='mso-bidi-font-weight:normal'><span lang=EN-US style='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'><!--[if gte vml 1]><v:shapetype id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f"> <v:stroke joinstyle="miter"/> <v:formulas>  <v:f eqn="if lineDrawn pixelLineWidth 0"/>  <v:f eqn="sum @0 1 0"/>  <v:f eqn="sum 0 0 @1"/>  <v:f eqn="prod @2 1 2"/>  <v:f eqn="prod @3 21600 pixelWidth"/>  <v:f eqn="prod @3 21600 pixelHeight"/>  <v:f eqn="sum @0 0 1"/>  <v:f eqn="prod @6 1 2"/>  <v:f eqn="prod @7 21600 pixelWidth"/>  <v:f eqn="sum @8 21600 0"/>  <v:f eqn="prod @7 21600 pixelHeight"/>  <v:f eqn="sum @10 21600 0"/> </v:formulas> <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/> <o:lock v:ext="edit" aspectratio="t"/></v:shapetype><v:shape id="_x0000_i1025" type="#_x0000_t75" style='width:678pt; height:325.2pt' filled="t"> <v:imagedata src="B_Deagle%202010_files/image008.png" o:title="Fig1sm"/></v:shape><![endif]--><![if !vml]><img border=0 width=904 height=434src="B_Deagle%202010_files/image001.gif" v:shapes="_x0000_i1025"><![endif]><o:p></o:p></span></b></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><b style='mso-bidi-font-weight:normal'><span lang=EN-USstyle='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></b></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><b style='mso-bidi-font-weight:normal'><span lang=EN-USstyle='font-size:16.0pt;font-family:Georgia;mso-ansi-language:EN-US'>(A)Phylogeography<o:p></o:p></span></b></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><span lang=EN-US style='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'><span style='mso-spacerun:yes'>�</span></span><spanstyle='font-size:14.0pt;font-family:Georgia'>Since phylogenetic hypothesesunderlie most conclusions in studies of adaptive evolution, a robustphylogeographic study in this model system would be invaluable. </span><spanlang=EN-US style='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'>Initialanalysis shows that populations from within a watershed cluster together andseparate watersheds are generally independent. A few cases of genetic linkagesbetween separate watersheds in adjoining geographical areas may reflect pastwatershed connections, or possibly ongoing gene flow.<b style='mso-bidi-font-weight:normal'> </b>Populations that do not group with others are either: a singlesampling site within a watershed, close to a river mouth (continued marine geneflow), or are isolated lakes with low heterozygosity which obscures historicalrelationships.</span><span lang=EN-US style='font-size:54.0pt;color:windowtext;mso-ansi-language:EN-US'><o:p></o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><span lang=EN-US style='font-size:14.0pt;color:windowtext;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><b style='mso-bidi-font-weight:normal'><span lang=EN-USstyle='font-size:16.0pt;font-family:Georgia;mso-ansi-language:EN-US'>(B) Parapatricstream/lake pairs of stickleback<o:p></o:p></span></b></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><span style='font-size:14.0pt;font-family:Georgia'>Parapatriclake and stream populations of stickleback are a classic example of ecologicalspeciation. Two watersheds on Haida Gwaii (Drizzle and Mayer) containpreviously described stream/lake pairs (</span><span style='font-size:10.0pt;font-family:Georgia'>Moodie 1972 </span><span lang=EN-US style='font-size:10.0pt;font-family:Georgia;mso-ansi-language:EN-US'>Can. J. Zool. 50: 721-732</span><spanstyle='font-size:10.0pt;font-family:Georgia'>; Reimchen <i>et al</i>. 1985 </span><spanlang=EN-US style='font-size:10.0pt;font-family:Georgia;mso-ansi-language:EN-US'>Can.J. Zool. 63:294-2951</span><span style='font-size:14.0pt;font-family:Georgia'>).</span><bstyle='mso-bidi-font-weight:normal'><span style='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'> </span></b><span style='font-size:14.0pt;font-family:Georgia'>To investigate the genetics underlying stream/lakedivergence we SNP genotyped fish from (1) <st1:PlaceName w:st="on">Drizzle</st1:PlaceName><st1:PlaceType w:st="on">Lake</st1:PlaceType> and Drizzle Outlet, and (2) <st1:placew:st="on"><st1:PlaceName w:st="on">Mayer</st1:PlaceName> <st1:PlaceType w:st="on">Lake</st1:PlaceType></st1:place>and two inlet streams (Gold and Woodpile).</span><span style='font-size:10.0pt;font-family:Georgia'><o:p></o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><span style='font-size:14.0pt;font-family:Georgia'>Dataconfirm independent origin of these stream/lake pairs in the separatewatersheds (Figure 2 ). </span><span lang=EN-US style='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'>Within the Mayer watershed, fishfrom two inlet creeks are closer to each other than to <st1:place w:st="on"><st1:PlaceName w:st="on">Mayer</st1:PlaceName> <st1:PlaceType w:st="on">Lake</st1:PlaceType></st1:place>fish. At the Gold Creek sampling site, both lake and stream forms werecollected and these samples are genetically distinct in sympatry. <o:p></o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><span style='font-size:14.0pt;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal align=center style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt;text-align:center'><span style='font-size:14.0pt;font-family:Georgia'><!--[if gte vml 1]><v:shape id="_x0000_i1026" type="#_x0000_t75" style='width:768.6pt;height:370.8pt'> <v:imagedata src="B_Deagle%202010_files/image002.png" o:title="Fig2"/></v:shape><![endif]--><![if !vml]><img border=0 width=1025 height=494src="B_Deagle%202010_files/image003.gif" v:shapes="_x0000_i1026"><![endif]><o:p></o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><span style='font-size:14.0pt;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><span style='font-size:14.0pt;font-family:Georgia'>Genomicdivergence between stream and lake forms within a watershed was highlyheterogeneous: most loci show moderate levels of divergence but some loci(outliers) were highly differentiated and are likely linked to genes understrong divergent selection (Figure 3). <o:p></o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><span style='font-size:14.0pt;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal align=center style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt;text-align:center'><span style='font-size:14.0pt;font-family:Georgia'><!--[if gte vml 1]><v:shape id="_x0000_i1038" type="#_x0000_t75" style='width:605.4pt;height:355.2pt' filled="t"> <v:imagedata src="B_Deagle%202010_files/image004.png" o:title="Fig4"/></v:shape><![endif]--><![if !vml]><img border=0 width=807 height=474src="B_Deagle%202010_files/image011.gif" v:shapes="_x0000_i1038"><![endif]><o:p></o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><span style='font-size:14.0pt;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><b><span lang=EN-US style='font-size:16.0pt;font-family:Georgia;mso-ansi-language:EN-US'>(C) </span></b><b style='mso-bidi-font-weight:normal'><span style='font-size:16.0pt;font-family:Georgia'>Associations betweenSNPs and morphological traits<o:p></o:p></span></b></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><span style='font-size:14.0pt;font-family:Georgia'>Populationsof Haida Gwaii stickleback exhibit a range of phenotypes defined by the location-specificselective landscapes (</span><span style='font-size:10.0pt;font-family:Georgia'>Moodieand Reimchen 1976 </span><span lang=EN-US style='font-size:10.0pt;font-family:Georgia;mso-ansi-language:EN-US'>Systematic Zoology 25: 49-61</span><spanstyle='font-size:14.0pt;font-family:Georgia'>). With morphological data fromover 100 populations (</span><span style='font-size:10.0pt;font-family:Georgia'>Reimchen<i style='mso-bidi-font-style:normal'>et al</i>. In prep</span><spanstyle='font-size:14.0pt;font-family:Georgia'>) and SNP data that characterizesindividuals from these populations, it may be possible to identify loci linkedto particular morphological traits (Figure 4).<o:p></o:p></span></p><p class=MsoNormal align=center style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt;text-align:center'><span style='font-size:14.0pt;font-family:Georgia'><!--[if gte vml 1]><v:shape id="_x0000_i1027" type="#_x0000_t75" style='width:721.8pt;height:330pt' filled="t"> <v:imagedata src="B_Deagle%202010_files/image007.png" o:title="Fig4"/></v:shape><![endif]--><![if !vml]><img border=0 width=962 height=440src="B_Deagle%202010_files/image009.gif" v:shapes="_x0000_i1027"><![endif]><o:p></o:p></span></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;margin-left:18.0pt'><b style='mso-bidi-font-weight:normal'><span lang=EN-USstyle='font-size:14.0pt;font-family:Georgia;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></b></p><p class=MsoNormal><span style='font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal><span style='font-family:Georgia'><o:p>&nbsp;</o:p></span></p><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div><div class=MsoNormal><span style='font-family:Georgia'><hr size=2 width="100%" align=left></span></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div><p class=MsoNormal align=center style='mso-margin-top-alt:auto;margin-bottom:12.0pt;text-align:center;text-indent:36.0pt'><u><span style='font-size:20.0pt;font-family:Georgia'>PhD work at the <ahref="http://www.zoo.utas.edu.au/awru/AWRU1020.htm">University of Tasmania</a>:DNA-based methods to study diet<o:p></o:p></span></u></p><p class=MsoNormal style='mso-margin-top-alt:auto;margin-bottom:12.0pt;text-indent:36.0pt'><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia'>Studying animal diet is a difficult undertaking especiallyin the marine environment where it is rarely possible to observe feedingactivity. Some prey remains survive digestion and morphological analysis ofremains in faecal material can provide an indication of what predators areeating. In my PhD I looked at the potential for use of DNA-based identificationmethods in dietary studies of marine predators. When I started it was clearthat prey DNA was present in animal faeces but how useful it would be fordietary studies was an open question. So, in collaboration with researchersfrom the <a href="http://www.marinemammal.org/MMRU/">Marine Mammal Unit at UBC</a>,I started by looking at samples collected from captive <ahref="http://www.marinemammal.org/steller_sea_lion/steller_sea_lion_fastfacts.php">Stellersea lions</a> to address some basic questions:<o:p></o:p></span></p><p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo2;tab-stops:list 18.0pt'><![if !supportLists]><spanstyle='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:Symbol;mso-bidi-font-family:Symbol'><spanstyle='mso-list:Ignore'>�<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span><![endif]><!--[if gte vml 1]><v:group id="_x0000_s1064" editas="canvas" style='position:absolute;left:0;text-align:left;margin-left:8473pt; margin-top:0;width:309pt;height:345pt;z-index:1;mso-position-horizontal:right; mso-position-vertical:top;mso-position-vertical-relative:line' coordorigin="7254,1434" coordsize="5150,5750" o:allowoverlap="f"> <o:lock v:ext="edit" aspectratio="t"/> <v:shape id="_x0000_s1063" type="#_x0000_t75" style='position:absolute;left:7254;  top:1434;width:5150;height:5750' o:preferrelative="f">  <v:fill o:detectmouseclick="t"/>  <v:path o:extrusionok="t" o:connecttype="none"/> </v:shape><v:shape id="_x0000_s1100" type="#_x0000_t75" style='position:absolute;  left:7970;top:1662;width:3703;height:4592'>  <v:imagedata src="B_Deagle%202010_files/image018.jpg" o:title="UBC FT"/> </v:shape><w:wrap type="square"/></v:group><![endif]--><![if !vml]><img width=412 height=460src="B_Deagle%202010_files/image012.gif" align=right v:shapes="_x0000_s1064 _x0000_s1063 _x0000_s1100"><![endif]><spanstyle='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Georgia'>Can prey DNA be reliably detected in the soft matrix of faecalsamples?</span><span style='color:windowtext;mso-fareast-language:EN-US'> </span><spanstyle='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Georgia'><o:p></o:p></span></p><p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo2;tab-stops:list 18.0pt'><![if !supportLists]><spanstyle='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:Symbol;mso-bidi-font-family:Symbol'><spanstyle='mso-list:Ignore'>�<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span><![endif]><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Georgia'>Can DNA from prey items fed as a smallproportion of the diet be detected?</span><span style='color:windowtext;mso-fareast-language:EN-US'> </span><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Georgia'><o:p></o:p></span></p><p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo2;tab-stops:list 18.0pt'><![if !supportLists]><spanstyle='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:Symbol;mso-bidi-font-family:Symbol'><spanstyle='mso-list:Ignore'>�<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span><![endif]><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Georgia'>Are the relative amounts of DNArecovered from prey species proportional to their mass in the diet?<o:p></o:p></span></p><p class=MsoNormal style='margin-left:18.0pt;text-indent:-18.0pt;line-height:150%;mso-list:l0 level1 lfo2;tab-stops:list 18.0pt'><![if !supportLists]><spanstyle='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Symbol;mso-fareast-font-family:Symbol;mso-bidi-font-family:Symbol'><spanstyle='mso-list:Ignore'>�<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></span></span><![endif]><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Georgia'>What is the quality of the preyDNA recovered?<o:p></o:p></span></p><p class=MsoNormal style='line-height:150%'><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='line-height:150%'><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Georgia'>These questionsare the focus of three papers from my thesis:<o:p></o:p></span></p><p class=MsoNormal style='line-height:150%'><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;line-height:150%;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'>DeagleBE, Tollit DJ, Jarman SN, Hindell MA, Trites AW, Gales NJ (2005) Molecularscatology as a tool to study diet: analysis of prey DNA in scats from captiveSteller sea lions. Molecular Ecology 14: 1831&#8211;1842. <ahref="http://www.marinemammal.org/pdfs/Deagle_2005.pdf">(pdf)</a><o:p></o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'>DeagleBE, Eveson JP, Jarman SN (2006) Quantification of damage in DNA recovered fromhighly degraded samples &#8212; a case study on DNA in faeces. Frontiers inZoology e3:11. <ahref="http://www.frontiersinzoology.com/content/pdf/1742-9994-3-11.pdf">(pdf)</a><o:p></o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'>DeagleBE, Tollit DJ (2007) Quantitative analysis of prey DNA in pinniped faeces:potential to estimate diet composition? Conservation Genetics 8: 743&#8211;747.<a href="http://www.marinemammal.org/pdfs/Deagle%20and%20Tollit%202007.pdf">(pdf)</a><o:p></o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:11.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:14.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:14.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:14.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia'>Themethods were further evaluated in a field-based study looking at the diet ofmacaroni penguins at <a href="http://www.heardisland.aq/">Heard Island</a>.Most recently I have been involved in projects using high-throughput sequencingto look at diet of Australian fur seals and little penguins: <o:p></o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:14.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'>DeagleBE, Gales NJ, Evans K, Jarman SN, Robinson S, Trebilco R, Hindell MA (2007)Studying seabird diet through genetic analysis of faeces: a case study onmacaroni penguins (<i style='mso-bidi-font-style:normal'>Eudyptes chrysolophus</i>).PLoS ONE 2: e831. <ahref="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0000831">(pdf)</a></span><spanstyle='font-size:10.0pt'> </span><span style='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'><o:p></o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:10.0pt'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'>DeagleBE,</span><span lang=EN-US style='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman;mso-ansi-language:EN-US'> Kirkwood R, JarmanSN (2009) Analysis of Australian fur seal diet by pyrosequencing prey DNA infaeces. Molecular Ecology 18: 2022&#8211;2038.<o:p></o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanlang=EN-US style='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman;mso-ansi-language:EN-US'>Deagle BE, Chiaradia A, McInnes J,Jarman SN (2010) </span><span style='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman;mso-bidi-font-weight:bold'>Pyrosequencingfaecal DNA to determine diet of little penguins: is what goes in what comesout?</span><span lang=EN-US style='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman;mso-ansi-language:EN-US'> ConservationGenetics 11: 2039&#8211;2048.<o:p></o:p></span></p><p class=MsoNormal style='mso-layout-grid-align:none;text-autospace:none'><spanstyle='font-size:10.0pt;font-family:TimesNewRoman;mso-bidi-font-family:TimesNewRoman'><o:p>&nbsp;</o:p></span></p><p align=center style='margin-bottom:12.0pt;text-align:center'><spanstyle='font-size:11.0pt;mso-bidi-font-size:12.0pt;font-family:Verdana'><o:p>&nbsp;</o:p></span></p><p align=center style='margin-bottom:12.0pt;text-align:center'><spanstyle='font-size:11.0pt;mso-bidi-font-size:12.0pt;font-family:Verdana'><spanstyle='mso-spacerun:yes'>��������������� </span><!--[if gte vml 1]><v:shape id="_x0000_i1028" type="#_x0000_t75" style='width:266.4pt;height:162.6pt'> <v:imagedata src="B_Deagle%202010_files/image020.jpg" o:title="Mac"/></v:shape><![endif]--><![if !vml]><img border=0 width=355 height=217src="B_Deagle%202010_files/image021.jpg" v:shapes="_x0000_i1028"><![endif]><spanstyle='mso-spacerun:yes'>����</span><!--[if gte vml 1]><v:shape id="_x0000_i1029" type="#_x0000_t75" style='width:285.6pt;height:163.8pt'> <v:imagedata src="B_Deagle%202010_files/image022.jpg" o:title="AusFS"/></v:shape><![endif]--><![if !vml]><img border=0 width=381 height=218src="B_Deagle%202010_files/image023.jpg" v:shapes="_x0000_i1029"><![endif]><o:p></o:p></span></p><p align=center style='margin-bottom:12.0pt;text-align:center'><spanstyle='font-size:11.0pt;mso-bidi-font-size:12.0pt;font-family:Verdana'><o:p>&nbsp;</o:p></span></p><p align=center style='margin-bottom:12.0pt;text-align:center'><spanstyle='font-size:11.0pt;mso-bidi-font-size:12.0pt;font-family:Verdana'><o:p>&nbsp;</o:p></span></p><p class=MsoNormal align=center style='mso-margin-top-alt:auto;margin-bottom:12.0pt;text-align:center'><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia'>For my other papers, please see <ahref="BD%20pub.pdf">publication list</a>.<o:p></o:p></span></p><p class=MsoNormal align=center style='mso-margin-top-alt:auto;margin-bottom:12.0pt;text-align:center'><span style='font-size:14.0pt;mso-bidi-font-size:12.0pt;font-family:Georgia'><o:p>&nbsp;</o:p></span></p><p align=center style='margin-bottom:12.0pt;text-align:center'><spanstyle='font-size:11.0pt;mso-bidi-font-size:12.0pt;font-family:Verdana'><o:p>&nbsp;</o:p></span></p></div></body></html>